### ISSUE TITLE
Allow turning off the mobile screen 

### ISSUE THREAD

**Issue by arky** [mapillary/mapillary_issues#1625]
Please allow turning off the screen to conserve battery. This is very useful feature when doing long term recording in a car.

**Comment by jrcarlsen** [comment]
It may also prevent burn on screens. I have the Mapillary crosshair permanently burned on my screen now.

**Comment by peterneubauer** [comment]
Ok,
That's a compelling reason. What mode would be best? Have a hardware button
or an on screen control? And how to wake it up again?

**Comment by kaitu** [comment]
Related to #68

**Comment by Caboosey** [comment]
Yes. Please give me this option or someway to have the ability hide the cross hairs. The Mapillary app has started to burn into my Samsung Galaxy S5 which has an OLED screen.

**Comment by mr-j0nes** [comment]
I see there is no more talks about this, but this is a feature that could be really make a huge difference.

Makes no sense to keep the display on once the app is configured to take pics automatically. Sort of when I setup my MP3 player I can just turn off the screen and the music keeps playing, if I want to access the player, I just turn on the screen and handle that from the notifications area, without unlocking or just by unlocking it.

I am not sure how the restrictions of Android are for using the camera when the phone is locked, I guess that might be the challenge ?

**Comment by arky** [comment]
@peterneubauer  Guess we now have enough feedback on this feature request. Perhaps it's time consider  adding this feature?

**Comment by mr-j0nes** [comment]
For those who suffer screen burns on Android I've just came across [blackr](https://play.google.com/store/apps/details?id=com.neximolabs.blackr&hl=en) that can be used as workaround. It does not turn the screen off but it blacks it out (For a complete black out hide the navigation..)

**Comment by katrinhumal** [comment]
Just to chime in - we are definitely considering this, the problem is just that the feature is competing with everything else that we need to work on with Android. If there was no resource issue, this would have likely been done long ago. :)

**Comment by ovivoda** [comment]
_mapillary_issues is no longer an active repository._

_Let’s continue discussing features in our forum! We will import this issue as topic to the <a href="http://forum.mapillary.com ">Mapillary Forum</a> in the coming weeks. We’ll notify here once it’s been done. See you there!_
